.github/workflows/flutter.yml

```
@@ -0,0 +1,208 @@
+    name: Setup Environment
+    runs-on: ubuntu-latest
+    outputs:
+      flutter_version: ${{ steps.flutter-version.outputs.FLUTTER_VERSION }}
+      coverage_excludes_regex_flags: ${{ steps.extract-config.outputs.COVERAGE_EXCLUDES_REGEX_FLAGS }}
+    steps:
+      - uses: actions/checkout@v3
+
+      # Extract Flutter version from pubspec.yaml
+      - name: Extract Flutter version
+        id: flutter-version
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1009:info:2:94: The mentioned syntax error was in this simple command

(shellcheck)